extension.css

```diff
@@ -21,25 +22,24 @@ body * {
   text-align: start;
 }
 
-.browser-style {
+.browser-style,
+.browser-style > input[type="text"] {
   -moz-appearance: none;
-  margin-bottom: 6px;
   text-align: left;
 }
 
 /* stylelint-disable property-no-vendor-prefix */
 /* Buttons */
 button.browser-style,
 select.browser-style {
-  background-color: #fbfbfb;
-  border: 1px solid #b1b1b1;
-  box-shadow: 0 0 0 0 transparent;
-  font: caption;
-  height: 24px;
+  background-color: var(--grey-90-a10);
+  border: 1px solid transparent;
+  border-radius: 2px;
+  font: inherit;
+  height: 32px;
+  margin-right: 16px;
   outline: 0 !important;
   padding: 0 8px 0;
-  transition-duration: 250ms;
-  transition-property: box-shadow, border;
```

Comment:
Are we really not transitioning these things anymore?  Huh.

Reply:
Yes, shorlander-approved.